packages/components/addon/components/hds/dropdown/toggle-user.js

```diff
@@ -34,7 +34,7 @@ export default class HdsDropdownToggleUserComponent extends Component {
    * @return {string} The "class" attribute to apply to the component.
    */
   get classNames() {
-    let classes = ['hds-dropdown-toggle', 'hds-dropdown-toggle--user'];
+    let classes = ['hds-dropdown-toggle', 'hds-dropdown-toggle-user'];
```

Comment:
[chore] same as above, if we use define this as a block, then the `hds-dropdown-toggle` should be removed.

Reply:
done